1-exercises/B-while-loop/exercise.js

```diff
@@ -7,8 +7,19 @@
 
 function evenNumbers(n) {
     // TODO
+if (n !== 0){
+    let i = 0; let p = 0; let numArray = [];
+    while (i < n) {
+        numArray.push(p);
+        i++;
+        p = (p + 2);
+    }
+    console.log(numArray.toString());
 }
```

Comment:
You seem to be overcomplicating this function. You should be able to just log the first n even numbers like so:
```
function evenNumbers(n) {
    let i = 0; let count = 0;
    while (count < n) {
      if (i % 2 === 0) {
        console.log(i)
        count++
      }
      i++
    }
}
```